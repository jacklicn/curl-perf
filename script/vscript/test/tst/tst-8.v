sub fib(a)
  if a <= 1 
    return a
  else 
    return fib(a-1) + fib(a-2)
  end
end


print( fib(5) )

var a = 3

while a < 10
  print( fib(a) )
  a = a + 1
end  

