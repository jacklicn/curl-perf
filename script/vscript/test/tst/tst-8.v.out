*** syntax ok ***
*** semantics and code generation ok ***

>5

>2

>3

>5

>8

>13

>21

>34
